plugin/coc-quick-lint-js/README.md

```diff
@@ -0,0 +1,15 @@
+# coc-quick-lint-js
+
+coc.nvim extension for quick-lint-js
+
+## Install
+
+`:CocInstall coc-quick-lint-js`
+
+## License
+
+MIT
```

Comment:
Blocking: Our code should be licensed under GPL v3 or newer, unless there's a good reason to license this code under an MIT-style license.